### PR TITLE
Adding  twin normative modelling project

### PR DESCRIPTION
**Pull request by likeajumprope**
Pull request to add twin normative modelling project to the website. 
Related to issue #149

### CODE REVIEW

File: content/en/project/Twin-normative-modelling/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2022-07-22" # Date you first upload your project.
+# Title of your project (we like creative title)
+title: "This is an example project page which serves as a template"
```
Comment (FrancoisPgm) [inline content/en/project/Twin-normative-modelling/index.md:5]:
```suggestion
title: "Twin normative modelling project"
```
put your project title here

File: content/en/project/Twin-normative-modelling/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+tags: [normative_modelling, twins, python, mri]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "In this project I am trying to train and test a normative model on a neuroimaging data set containing twin longitudinal data. I want to look at both changes of deviations in  z-scores over time and differences in z-scores between twins "
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+
+<img src="images/normative.png" width="400">
```
Comment (FrancoisPgm) [inline content/en/project/Twin-normative-modelling/index.md:27]:
```suggestion
image: "images/normative.png"
```

**Review by FrancoisPgm — CHANGES_REQUESTED** [review]
The title and cover image were not properly set but other than that it's great.

**Comment by likeajumprope** [comment]
I have committed the suggested changes - what else needs to be done so that the PR can be merged? :)

**Comment by pbellec** [comment]
Looks great @likeajumprope sorry about the delay! 

Your project is going to be incorporated in the 2023 gallery, because we took so long to merge it.

**Pull request merged**